beacon-chain/core/electra/transition_no_verify_sig.go

```diff
@@ -84,14 +85,20 @@ func ProcessOperations(
 	}
 	st, err = ProcessDepositRequests(ctx, st, requests.Deposits)
 	if err != nil {
-		return nil, errors.Wrap(err, "could not process deposit requests")
+		if errors.Is(err, errNilExecutionRequest) {
```

Comment:
no logs required here? because they are processed inside?

Reply:
it's logged inside `ProcessDepositRequests`